.github/CHANGELOG.md

```diff
@@ -2,6 +2,63 @@
 
 <h3>New features since last release</h3>
 
+* Two new quantum function transforms have been added to enable the
+  removal of redundant gates in quantum circuits.
+  [(#1455)](https://github.com/PennyLaneAI/pennylane/pull/1455)
+
+  The ``cancel_inverses`` transform loops through a list of operations,
```

Comment:
```suggestion
  The `cancel_inverses` transform loops through a list of operations,
```

Reply:
Ugh I keep making this mistake, there are some places where it *does* have to be two ` instead of just one, right?

Reply:
Yep! ReST files/docstrings use double backticks, markdown uses single backticks